weltyt.py

```diff
@@ -322,6 +340,30 @@ def buy_lands(lands):
 
     print("Go forward")
 
+    planted_lands = []
+    for p in cur.execute("SELECT * FROM lands WHERE plants>0"):
```

Comment:
You can use list comprehension for that.

Reply:
`planted_lands = [p for p in cur.execute("SELECT * FROM lands WHERE plants>0")]` doesn't work?

Reply:
Unfortunately, then planted_lands.append(p) doesn't work, and NameError occurs.

Reply:
Why do you want to append anything? The point is you don't have to - list comprehension creates the list for you.